### DISCUSSION TITLE
Temperatur der Batterie über MQTT auslesen

### DISCUSSION THREAD

**Discussion by flurfunk0868** [hoylabs/OpenDTU-OnBattery#606] [Q&A] [answered]
Hallo zusammen,

mein Victron Laderegler holt sich die Batterie Temperatur fürs Laden/Entladen vom Victron Smart Battery Sense (Verbindung über das integrierte Bluetooth Netzwerk). Da der Laderegler ja schon in Home Assistant über MQTT eingepflegt ist, wollte ich wissen ob man dann auch die Temperatur übergeben bzw. abfragen kann? Leider ist die bei den ganzen Entitäten bislang nicht dabei.

Viele Grüße

**Comment by schlimmchen** [comment, marked as answer]
Servus @flurfunk0868, das ist leider nicht ohne erhöhten Aufwand machbar. Es mag sein, dass dein Laderegler die Temperatur der Batterie kennt, aber er wird sie nicht in seiner Standard-TEXT-Nachricht über UART übermitteln. Man könnte diesen Wert vielleicht aktiv abfragen über das HEX-Protokoll, allerdings ist weder das Senden von Daten an den Laderegler noch das HEX-Protokoll zur Zeit unterstützt.

Vermutlich hast du deine Batterie schon am laufen, also hast auch ein integriertes oder externes BMS bereits angeschafft. Dennoch der Hinweis aufs [JK BMS](https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Jikong-JK-BMS), welches auch die Batterietemperatur meldet.

**Reply by flurfunk0868** [reply]
Hi, danke für die Rückmeldung. Wäre zwar "nice to have" gewesen aber dafür lohnt sich wahrscheinlich der Aufwand nicht. Meine Batterie hat zwar ein integriertes BMS aber ich traue dem Ding nicht zu 100%, deswegen lasse ich meine Batterie mit dem Smart-Sense überwachen und verhindere so zB. das Laden unter 2 °C. Ein externes BMS ist bei mir momentan nicht geplant, aber gut zu wissen dass es sowas gibt.
